src/components/delegates/DelegatesTable.tsx

```diff
@@ -62,6 +62,7 @@ export type DelegatesTableProps = {
 	initialSearch?: string;
 	address?: string;
 	download?: boolean;
+	fromValidator?: boolean;
```

Comment:
i don't like this

Reply:
Do you want another component for DelegatesTable in Validator page?